docs/terms/datasource.md

```diff
@@ -50,14 +50,9 @@ Datasources support connecting to a variety of different data backends.  No matt
 
 Datasources do not modify your data during profiling or validation, but they may create temporary artifacts to optimize computing Metrics and Validation
```

Comment:
Would it be useful to note here that it's possible to control whether or not GE creates temporary artifacts? Something like:
```
Datasources do not modify your data during profiling or validation, but they may create temporary artifacts to optimize computing Metrics and Validation. 
This behavior can be configured at the Data Connector level.
```

Reply:
Added the requested note.